modules/mcuboot/Kconfig

```diff
@@ -112,4 +112,13 @@ config ADD_MCUBOOT_MEDIATE_SIM_FLASH_DTS
 	  multi image update of the network core or to allow serial recovery of
 	  the network core.
 
+config MCUBOOT_USE_ALL_AVAILABLE_RAM
+	bool "Allow MCUBoot to use all available RAM (security implications)"
+	depends on TRUSTED_EXECUTION_NONSECURE || TRUSTED_EXECUTION_SECURE
+	default y if BOARD_THINGY53_NRF5340_CPUAPP_NS
```

Comment:
You can't build mcuboot for a non secure target so I'm not really sure what the point of this is

Reply:
True enough. To be honest I wanted a option to tell `partition_manager` that it should `USE_ALL_AVAILABLE_RAM` for MCUBoot child_image. I just made the change without thinking properly thanks for pointing it out. I'll double check if this actually work as intended or not.

Reply:
This works as intended. This Kconfig is for configuring the mcuboot module from the parent image.

Reply:
With regards to your PCD question
```
  sram_primary (0x80000 - 512kB):
+------------------------------------------------+
| 0x20000000: pcd_sram (0x2000 - 8kB)            |
+---0x20002000: mcuboot_sram (0x7e000 - 504kB)---+
+---0x20002000: sram_secure (0x8000 - 32kB)------+
| 0x20002000: tfm_sram (0x8000 - 32kB)           |
+---0x2000a000: sram_nonsecure (0x76000 - 472kB)-+
| 0x2000a000: sram_primary (0x66000 - 408kB)     |
| 0x20070000: rpmsg_nrf53_sram (0x10000 - 64kB)  |
+------------------------------------------------+
```

Hope this clears up your confusion.